Remotion/Development/UnitTests/Web/UnitTesting/UI/Controls/Rendering/HtmlHelperBaseTest.cs

```diff
@@ -124,6 +124,16 @@ public void Succeed_GetResultDocument ()
       Assert.That(document.DocumentElement.Name, Is.EqualTo("TopLevelElement"));
     }
 
+    [Test]
+    public void Fail_GetResultDocument ()
+    {
+      _htmlHelper.Writer.Write("randomError");
+
+      Assert.That(
+          () => _htmlHelper.GetResultDocument(),
+          Throws.Exception.With.Message.EqualTo("Data at the root level is invalid. Line 1, position 665.\r\n\r\nContent:\r\n" + c_document + "randomError"));
```

Comment:
```suggestion
          Throws.Exception.OfType<XmlException>().With.Message.EqualTo("Data at the root level is invalid. Line 1, position 665.\r\n\r\nContent:\r\n" + c_document + "randomError").And.With.InnerException.IsNull);
```

Note: I think the part with the InnerExcpetion is possible, but I did not acutally try it. If it's not a perfect match, please verify syntax and if it appears to not work at all with the single-line syntax while remaining type safe, we leave it out. The general idea is to document that we intentionally skipped the inner exception

Reply:
I don't need to recheck it but perhaps ask @Frazerus for a once over when you updated to make sure I didn't miss anything?